requirements/locks/py310-lock-linux-64.txt

```diff
@@ -328,5 +328,5 @@ https://conda.anaconda.org/conda-forge/noarch/ipywidgets-8.0.7-pyhd8ed1ab_0.cond
 https://conda.anaconda.org/conda-forge/noarch/nbconvert-pandoc-7.6.0-pyhd8ed1ab_0.conda#e8172ca42f2869bb90185c9356899e81
 https://conda.anaconda.org/conda-forge/linux-64/vtk-9.2.6-qt_py310h1234567_208.conda#934f7c25a6cf4cbd55c04e352ff7869a
```

Comment:
Bliss. No change :+1: 

#379 purged `vtk` as a dependency, and yet `conda-lock` has been influence to resolve to the `qt` rather than the `osmesa` build of `vtk` with the headless GH runner.

Sweet. I've been looking to resolve this issue for quite a wee while. Thanks for forcing it to a head @banesullivan. I think we're good now :+1: